Sources/PersistentCollections/PersistentDictionary._Node.swift

```diff
@@ -51,32 +51,32 @@ extension PersistentDictionary {
     var header: _NodeHeader
     var count: Int
 
-    let dataCapacity: Capacity
-    let trieCapacity: Capacity
+    let itemCapacity: Capacity
+    let childCapacity: Capacity
 
-    let dataBaseAddress: UnsafeMutablePointer<Element>
-    let trieBaseAddress: UnsafeMutablePointer<_Node>
+    let itemBaseAddress: UnsafeMutablePointer<Element>
+    let childBaseAddress: UnsafeMutablePointer<_Node>
 
     deinit {
-      dataBaseAddress.deinitialize(count: header.dataCount)
-      trieBaseAddress.deinitialize(count: header.trieCount)
+      itemBaseAddress.deinitialize(count: header.itemCount)
+      childBaseAddress.deinitialize(count: header.childCount)
```

Comment:
It could go either way, I don't much mind. The plural is more korrekt (if we want to be pedantic, which I don't really care about), but it's also more verbose. Consistency with `items`/`children` is attractive.

Looking at these now, the `BaseAddress`/`BasePointer` suffix seems overly ceremonious, though. I think I'll just call these `itemsStart`/`childrenStart`.

Reply:
Ah, then again, `itemCount` and `childCount` are better names than `itemsCount` and `childrenCount`. \<Annoyed noises>